File: URI/1079.js
var input = require('fs').readFileSync('./dev/stdin/arquivo.txt', 'utf8');
var lines = input.split('\n');

let vezes = +lines.shift();
const pegarValores = (line) => line.split(" ").map(a => +a);
let soma = 0;
for (let i = 0;i < vezes; i++){
    let notas = pegarValores(lines.shift()); 
    for (let j in notas){
        if (j === 0){
            soma = (notas[j]*2)
        }else if (j === 1){
            soma += (notas[j]*3)
        }else{
            soma += (notas[j]*5)
        }
    }
    j = 0;
    notas = 0;
    soma = 0;
}

File: URI/concluidos/Python/1079.py
casos = int(input())

while casos > 0:
    floatList = [float(item) for item in input().split(" ")]
    print('%0.1f'% (((floatList[0] * 2) + (floatList[1] * 3) + (floatList[2] * 5)) / 10))
    casos -= 1
